checkov/terraform/checks/resource/gcp/GoogleComputeInstanceOSLogin.py

```diff
@@ -7,7 +7,7 @@ def __init__(self):
         name = "Ensure that no instance in the project overrides the project setting for enabling OSLogin" \
                "(OSLogin needs to be enabled in project metadata for all instances)"
         id = "CKV_GCP_34"
-        supported_resources = ['google_compute_instance']
+        supported_resources = ['google_compute_instance', 'google_compute_instance_template']
```

Comment:
```suggestion
        supported_resources = ['google_compute_instance', 'google_compute_instance_template', 'google_compute_instance_from_template']
```
because it is a negative check